copy/copy.go

```diff
@@ -256,13 +259,14 @@ func Image(ctx context.Context, policyContext *signature.PolicyContext, destRef,
 	copyInParallel := dest.HasThreadSafePutBlob() && rawSource.HasThreadSafeGetBlob()
 
 	c := &copier{
-		dest:             dest,
-		rawSource:        rawSource,
-		reportWriter:     reportWriter,
-		progressOutput:   progressOutput,
-		progressInterval: options.ProgressInterval,
-		progress:         options.Progress,
-		copyInParallel:   copyInParallel,
+		dest:                 dest,
+		rawSource:            rawSource,
+		reportWriter:         reportWriter,
+		progressOutput:       progressOutput,
+		maxParallelDownloads: options.MaxParallelDownloads,
+		progressInterval:     options.ProgressInterval,
+		progress:             options.Progress,
+		copyInParallel:       copyInParallel,
```

Comment:
Please keep the field order in constructor the same as in the type definition. (I don’t care which which of the two, is updated, or perhaps both.)

Reply:
Fixed